packages/interpreter/src/sledgehammer_bindings.rs

```diff
@@ -146,6 +146,7 @@ mod js {
         controls: true,
         default: true,
         defer: true,
+        directory: true,
```

Comment:
The names in the boolean attributes are the final string names, not the rust attribute names because the interpreter will receive the attribute "webkitdirectory" instead of the "directory".  Can you change this to webkitdirectory: true

We also need to change the boolean attributes in the [common.js](https://github.com/DioxusLabs/dioxus/blob/master/packages/interpreter/src/common.js#L1) file to match the changes you made here.